util/src/main/java/org/killbill/billing/util/export/dao/DatabaseExportDao.java

```diff
@@ -166,14 +166,21 @@ private void exportDataForAccountAndTable(final DatabaseExportOutputStream out,
                     tableType = TableType.NOTIFICATION;
                 }
             }
+            if(tableType == TableType.EXTRA && column.getColumnName().equalsIgnoreCase(TableType.NOTIFICATION.getAccountRecordIdColumnName())) { //for aviate_notifications
```

Comment:
Just a thought: If we were to change line 161 and handle both `if (tableType == TableType.OTHER || tableType == TableType.EXTRA )`, wouldn't we not get this for free, essentially allowing any `notifications` table from any plugins that is using notifications tables (e.g. analytics) ?

Reply:
Addressed in 1c08594